locator/network_topology_strategy.cc

```diff
@@ -510,6 +509,26 @@ future<tablet_replica_set> network_topology_strategy::add_tablets_in_dc(schema_p
     co_return replicas;
 }
 
+tablet_replica_set network_topology_strategy::drop_tablets_in_dc(schema_ptr s, const locator::topology& topo,
```

Comment:
"s" is unused.

Reply:
The plan was to use it to print keyspace.table.  I'll add that use.

Reply:
used in v4